### DISCUSSION TITLE
Вопрос о Deployments в проекте GitHub

### DISCUSSION THREAD

**Discussion by bebka16** [community/community#132565] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Здравствуйте! У меня есть репозиторий GitHub https://github.com/DecibelProject/Decibel-Note , и в начале это был репозиторий типа GitHub Pages, но потом я решил изменить этот репозитоорий на обычный. Но проблема в том, что на главной странице репозитория есть пункт "Deployments", который хочется убрать, но у меня никак не получилось его удалить и сделать проект обычным. Как это сделать? decibelmeta@gmail.com

**Comment by danialtavakoli** [comment]
Здравствуйте!

Чтобы удалить раздел "Deployments" с главной страницы репозитория на GitHub и сделать его обычным репозиторием, нужно отменить публикацию GitHub Pages. Вот шаги, которые помогут вам это сделать:

1. **Перейдите в настройки репозитория**:
    - Откройте ваш репозиторий на GitHub.
    - Перейдите в раздел "Settings" (находится в верхнем меню репозитория).

2. **Найдите настройки GitHub Pages**:
    - Прокрутите вниз до секции "Pages" или "GitHub Pages". Эта секция может быть в нижней части страницы настроек.

3. **Отключите GitHub Pages**:
    - В секции "Source" выберите "None" (или "Disable GitHub Pages").
    - Сохраните изменения.

4. **Проверьте главную страницу репозитория**:
    - Перейдите обратно на главную страницу репозитория и убедитесь, что раздел "Deployments" больше не отображается.

Если эти шаги не решают проблему, можно попробовать следующий подход:

1. **Переименование ветки**:
    - Убедитесь, что текущая ветка `main` не используется для GitHub Pages.
    - Переименуйте основную ветку (например, из `gh-pages` в `main`, если она была настроена для GitHub Pages). Это можно сделать через командную строку или через интерфейс GitHub.

    ```bash
    git branch -m gh-pages main
    git push origin main
    ```

2. **Удаление ветки `gh-pages` (если она есть)**:
    - Убедитесь, что ветка `gh-pages` больше не нужна, и удалите ее.

    ```bash
    git push origin --delete gh-pages
    ```

Эти шаги должны помочь вам сделать репозиторий обычным и удалить раздел "Deployments". Если у вас возникнут дополнительные вопросы или проблемы, дайте знать!

**Comment by LiteBrite82** [comment]
Hi @DecibelProject , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
